database/src/test/scala/za/co/absa/atum/database/runs/CreatePartitioningIfNotExists.scala

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.database.runs
+
+import za.co.absa.balta.DBTestSuite
+import za.co.absa.balta.classes.JsonBString
+
+class CreatePartitioningIfNotExists extends DBTestSuite{
+
```

Comment:
What if both partitioning and parentPartitiining are created in one function call?
I do not see a status code for that. Is that a valid scenario?

If it is:
- then there will be a lost first set of status to 11 and will be replaced by setting 12.